File: platform/sdk/nodejs/examples/cloud/cloud-020_quota_burst_10.js
const { CloudClient } = require("../../mox-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
const result = client.quotaBurst10(15);
if (!result.throttled) process.exit(1);
console.log("XJ-OK: cloud-020_quota_burst_10");
process.exit(0);

File: platform/sdk/python/examples/cloud/cloud-020_quota_burst_10.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
result = client.quotaBurst10(15)
if not result["throttled"]:
    sys.exit(1)
print("XJ-OK: cloud-020_quota_burst_10")
sys.exit(0)
